news/2019-02-19-new-featured-artist-tieff.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "New Featured Artist: tieff"
+date: 2019-02-19 08:00:00 +0000
+---
+
+Bringing a founding member of the early osu! mapping scene back into the limelight in a whole new way, we're proud to announce our latest addition to the Featured Artist roster - tieff!
+
+[![](https://assets.ppy.sh/artists/34/header.jpg)](https://osu.ppy.sh/beatmaps/artists/34)
+
+Here since almost the very beginning, just about everyone with even a passing knowledge of osu!'s history will know **tieff**'s  name as one of the old 'great' mappers who laid out the creative foundation that we now all take for granted.
```

Comment:
```suggestion
Here since almost the very beginning, just about everyone with even a passing knowledge of osu!'s history will know **tieff**'s  name as one of the old 'great' mappers who laid out the creative foundations that we now all take for granted.
```